week-4/Homework/mandatory/3-freecodecamp.md

```diff
@@ -0,0 +1,31 @@
+## FreeCodeCamp Lessons
```

Comment:
As discussed let's move those to extras directory, as the syllabus doesn't include a lot of what the tutorials will be talking about.